.taskcluster.yml

```diff
@@ -197,142 +198,156 @@ tasks:
+                  created: {$fromNow: ''}
+                  deadline: {$fromNow: '1 day'}
+                  expires: {$fromNow: '1 year 1 second'}  # 1 second so artifacts expire first, despite rounding errors
+                  metadata:
+                    $merge:
+                      - owner: "${ownerEmail}"
+                        source: '${repoUrl}/raw/${head_sha}/.taskcluster.yml'
+                      - $if: 'tasks_for in ["github-push", "github-pull-request", "github-release"]'
+                        then:
+                          name: "Decision Task"
+                          description: 'The task that creates all of the other tasks in the task graph'
+                  provisionerId: "app-services-${level}"
+                  workerType: "decision"
+                  tags:
+                    $if: 'tasks_for in ["github-push", "github-pull-request"]'
                     then:
-                      - 'assume:repo:${repoUrl[8:]}:release'
-
-              requires: all-completed
-              priority: lowest
-              retries: 5
-
-              payload:
-                env:
-                  # run-task uses these to check out the source; the inputs
-                  # to `mach taskgraph decision` are all on the command line.
-                  $merge:
-                    - APPSERVICES_BASE_REPOSITORY: '${baseRepoUrl}'
-                      APPSERVICES_HEAD_REPOSITORY: '${repoUrl}'
-                      APPSERVICES_HEAD_REF: '${head_branch}'
-                      APPSERVICES_HEAD_REV: '${head_sha}'
-                      APPSERVICES_REPOSITORY_TYPE: git
-                      TASKGRAPH_BASE_REPOSITORY: https://hg.mozilla.org/ci/taskgraph
-                      TASKGRAPH_HEAD_REPOSITORY: https://hg.mozilla.org/ci/${taskgraph.branch}
-                      TASKGRAPH_HEAD_REV: ${taskgraph.revision}
-                      TASKGRAPH_REPOSITORY_TYPE: hg
-                      REPOSITORIES: {$json: {appservices: "Application Services", taskgraph: "Taskgraph"}}
-                      HG_STORE_PATH: /builds/worker/checkouts/hg-store
-                      ANDROID_SDK_ROOT: /builds/worker/android-sdk
-                      MOZ_FETCHES_DIR: /builds/worker/checkouts/src
-                    - $if: 'tasks_for in ["github-pull-request"]'
+                      kind: decision-task
+                  routes:
+                    $flattenDeep:
+                      - checks
+                      - $if: 'level == "3"'
+                        then:
+                          - tc-treeherder.v2.${project}.${head_sha}
+                          # TODO Bug 1601928: Make this scope fork-friendly once ${project} is better defined. This will enable
+                          # staging release promotion on forks.
+                          - $if: 'tasks_for == "github-push"'
+                            then:
+                              - index.project.${project}.v2.branch.${short_head_branch}.latest.taskgraph.decision
```

Comment:
@JohanLorenzo started curating these indexes in https://bugzilla.mozilla.org/show_bug.cgi?id=1631839 but I'll address that in a follow-up once releases are ready, to not add another possible source of failure in this long-overdue migration.